specificatie/openapi.yaml

```diff
@@ -558,6 +683,43 @@ components:
         pattern: '^[0-9]{16}$'
         example: '0226010000038820'
 
+    woonplaatsIdentificatie-query:
+      description: "De identificatie van een woonplaats uit de BAG. Deze is 4 cijfers lang."
+      name: woonplaatsIdentificatie
+      in: query
+      required: false
+      schema:
+        type: string
+        pattern: '^[0-9]{3}[1-9]|[0-9]{2}[1-9][0-9]|[0-9][1-9][0-9]{2}|[1-9][0-9]{3}$'
+        example: '2096'
+
+    geconstateerd:
+      description: "Een aanduiding waarmee kan worden aangegeven dat een object in de registratie is opgenomen als gevolg van een feitelijke constatering, zonder dat er op het moment van opname sprake was van een regulier brondocument voor deze opname."
+      name: geconstateerd
+      in: query
+      required: false
+      schema:
+        type: boolean
+        
+    type:
+      description: "Een aanduiding waarmee kan worden aangegeven dat een object in de registratie is opgenomen als gevolg van een feitelijke constatering, zonder dat er op het moment van opname sprake was van een regulier brondocument voor deze opname."
+      name: type
+      in: query
+      required: false
+      schema:
+        $ref: '#/components/schemas/TypeAdresseerbaarObject_enum'
+
+    gebruiksdoel:
+      description: "Een aanduiding waarmee kan worden aangegeven dat een object in de registratie is opgenomen als gevolg van een feitelijke constatering, zonder dat er op het moment van opname sprake was van een regulier brondocument voor deze opname."
+      name: gebruiksdoel
```

Comment:
moet zijn: "gebruiksdoelen", want is array (dus meervoud) én property in de resource heet ook "gebruiksdoelen"